src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.Android.cs

```diff
@@ -144,6 +149,12 @@ private static TimeZoneInfo GetLocalTimeZoneCore()
 
         private static TimeZoneInfoResult TryGetTimeZoneFromLocalMachineCore(string id, out TimeZoneInfo? value, out Exception? e)
         {
+            if (Invariant)
+            {
+                value = null;
+                e = new TimeZoneNotFoundException(SR.Format(SR.InvalidTimeZone_InvalidId, id));
+                return TimeZoneInfoResult.TimeZoneNotFoundException;
```

Comment:
can this codepath be hit when I want to lookup the UTC timezone? I assume we want that to work right?

Reply:
UTC is not loaded from OS, but rather created with zero `BaseUtcOffset`

https://github.com/dotnet/runtime/blob/c5bb5057dddf4b5834626e90e63bfd50f0cda3ff/src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.cs#L2216-L2221